src/test/kerberos/t/001_auth.pl

```diff
@@ -25,6 +25,10 @@
 {
 	plan skip_all => 'GSSAPI/Kerberos not supported by this build';
 }
+elsif ($ENV{PG_TEST_EXTRA} !~ /kerberos/)
+{
+	plan skip_all => 'GSSAPI/Kerberos is not enabled';
```

Comment:
Maybe reference PG_TEST_EXTRA? The message makes it sound like the whole feature isn't enabled.